app/dcs/third-party-tools/bunny/page.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Bunny CDN
+docId: pheaSeeSh5Eepee0ooth3
+metadata:
+  description:
+    Learn how to use Storj as the origin service for Bunny CDN.
+  title: 'Using Storj with Bunny CDN'
+---
+
+Bunny CDN is a content delivery network that works well with Storj, providing a caching layer beyond Storj's already relaible global performance.
+Storj is great for distributing files around the work, but for some workloads a Content Delivery Network is the more appropriate choice.
+CDNs leverage distributed caching to reduce latency and enhance website performance from servers located closer to the end-users.
+
+The following directions show how to set up static website hosting with Storj and Bunny CDN.
+
+
+## Create a publicly shared bucket / prefix in Storj
+
+To work with Bunny CDN, you'll first need to share a bucket or prefix (folder) in Storj.
+
+This can be done by following Part 1 of our [](docId:GkgE6Egi02wRZtyryFyPz) directions, or using the Storj.io's 
+[](docId:4oDAezF-FcfPr0WPl7knd#share-a-bucket).
+
+If using our CLI tool, record the `storj-root` and `storj-access` values from the [](docId:GkgE6Egi02wRZtyryFyPz) directions.
+
+If using the Storj.io website, you'll need to determine the `storj-root` and `storj-access` values from the Shared Link.
+Visit the Shared Link in a browser, and copy the new, resulting URL.
+This URL will follow a known formula:  `https://link.storjshare.io/s/` + `storj-access` + `/` + `storj-root`.
+
+In the example of `https://link.storjshare.io/s/jwm2j2juznpzcyl5kzgxzxvareoq/mybucket/myfolder/`, 
+record `storj-root:jwm2j2juznpzcyl5kzgxzxvareoq` and `storj-access:/mybucket/myfolder/`
+
+{% callout type="info"  %}
+The `storj-root` part of a Shared Link URL is "URL-encoded."
+URL-encoded values often have slash characters ('/') replaced with `%2F`.
+Visiting the URL in a browser is a convenient fix in most circumstances.
+{% /callout %}
+
+
+## Configure Bunny CDN
+
+Log into Bunny.
```

Comment:
You can prefix each of these steps with `1.` and it'll make a nice numbered list for the reader